examples/tests/sdk_tests/api/delete.w

```diff
@@ -0,0 +1,37 @@
+bring cloud;
+bring http;
+
+let http_DELETE = http.HttpMethod.DELETE;
```

Comment:
Why is this needed?

Reply:
https://winglang.slack.com/archives/C048QCN2XLJ/p1687425563136599
Opened a bug: https://github.com/winglang/wing/issues/3049
I'll add the link to those files